### DISCUSSION TITLE
principle of operation of the H-Hope current meter

### DISCUSSION THREAD

**Discussion by prvinavasi** [H-HOPE/H-HOPE-current-meter#3] [General]
I wonder how the H-Hope current meter works, is it similar to ship propeller operation?
Thank you for your answer.

**Comment by H-HOPE** [comment]
Thank you for your question,

a ship propeller converts engine power into thrust to move a vessel through water, typically featuring multiple twisted blades for efficient propulsion. The H-HOPE water current meter while rotating does not interact with water, producing no vorticies and therefore does not exchange energy with the flow. As a result the frequency of rotation is propotional to the water flow velocity. The Arduino microcontroller measures the frequency of rotation and displays the velocity on the screen.
